zadani/soft/style.css

```diff
@@ -11,33 +13,166 @@
 body {
     margin: 0;
     padding: 0;
-    background-color: #f5f5f5;
     font-family: 'Merriweather', serif;
     font-size: 16px;
     line-height: 1.6;
+    background-color: #f5f5f5;
 }
 
-
 /* HLAVICKA */
 
+.hlavicka {
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    background-image: url('images/hlavicka.jpg');
+    /* proc se nezobrazuje???*/
+    min-width: 500px;
+}
+
 .titulek {
     font-family: 'Raleway', serif;
+    font-size: 3rem;
+}
+
+.autor {
+    font-style: italic;
+    border-top: 5px solid white;
+    padding: 1rem;
 }
 
 
 /* MENU */
 
+.menu-obsah {
+    display: flex;
+    flex-direction: column;
+    min-width: 500px;
+    background-color: white;
+    box-shadow: 5px 10px 18px grey;
+}
+
+ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+}
+
+.menu {
+    text-align: center;
+    list-style-type: none;
+    text-transform: uppercase;
+}
 
+.menu a:link {
+    text-decoration: none;
+    color: gray;
+}
 
+a:visited {
+    text-decoration: none;
+    color: gray;
+}
 
-/* OBSAH */
+.polozka {
+    display: flex;
+    flex-direction: column;
+    padding: 0.5rem;
+    margin: 1.5rem;
+}
 
+p {
+    padding-left: 1rem;
+    padding-right: 1rem;
+}
 
 
+/* OBSAH */
+
+a:link {
+    color: gray;
+}
+
+.popisek {
+    font-size: small;
+    justify-content: center;
+}
 
 /* FOTOGALERIE */
 
+.foto {
+    display: flex;
+    flex-direction: column;
+}
 
+.popisek {
+    font-weight: bold;
+    padding-bottom: 1rem;
+    text-align: center;
+    color: grey;
+}
 
+.popisek-copy {
+    font-weight: normal;
+}
 
 /* PATICKA */
+
+.paticka {
+    display: flex;
+    flex-flow: column;
+    font-size: small;
+    font-weight: bold;
+    color: grey;
+    min-width: 500px;
+}
+
+
+/* TABLET */
+
+@media (min-width: 501px) {
```

Comment:
Tady by mělo být správně 500, ale tobě to tu komplikuje to, že sis všude nastavila 500 jako minimální šířku.